update the pages for 22.04 LTS support (#25)
NLnetLabs/unbound-manual: source/getting-started/configuration.rst

Then Unbound will fork to the background and run until you either kill it or
reboot the machine.

You may run into an error where Unbound tells you it cannot bind to the 
``0.0.0.0`` as it's already in use. This is because the system resolver 
``systemd-resolved`` is already running on that port. You can go around this by

> changing the the IP address in the config to ``127.0.0.1``. This looks like:

Reviewer: ```suggestion
changing the IP address in the config to ``127.0.0.1``. This looks like:
```

.. code-block:: bash

    server:
        # specify the interface to answer queries from by ip-address.
        interface: 127.0.0.1

> If you want to change this behavior, on :doc:`this page</use-cases/local-stub>`

Reviewer 2: ```suggestion
If you want to change this behaviour, on :doc:`this page</use-cases/local-stub>`
```

To help, we can use the :doc:`/manpages/unbound-anchor` command.

:command:`unbound-anchor` performs the setup by configuring a trust anchor. This

> trust anchor will only serve as the initial anchor from builtin values. To keep

Reviewer 2: ```suggestion
trust anchor will only serve as the initial anchor from built-in values. To keep
```
